src/main/java/banking/BankingSystem.java

```diff
@@ -0,0 +1,200 @@
+            case "0" -> System.out.println("\nBye!\n");
+            default -> System.out.println("Incorrect option. Try again.");
+        }
+    }
+
+    private void showBalanceOption() {
+        System.out.println("Balance: " + currentAccount.balance());
+    }
+
+    private void addIncomeOption() {
+        System.out.println("\nEnter income:");
+
+        try {
+            int income = scanner.nextInt();
+            if (income < 1) {
+                throw new Exception();
+            }
+
+            bank.addIncome(currentAccount.card().cardNumber(), income);
+            System.out.println("Income was added!");
+        } catch (Exception ex) {
+            System.out.println("Incorrect input. Enter a number greater than zero!");
+        }
+    }
+
+    private void doTransferOption() {
+        System.out.println("\nTransfer");
+        System.out.println("Enter card number:");
+
+        try {
+            String receiver = scanner.next();
+            if (receiver.equals(currentAccount.card().cardNumber())) {
+                throw new Exception("You can't transfer money to the same account!");
+            }
+            bank.checkCardNumber(receiver);
+
+            System.out.println("Enter how much money you want to transfer:");
+            int moneyToTransfer = scanner.nextInt();
+
+            if (moneyToTransfer < 1) {
+                throw new IllegalArgumentException();
+            } else if (moneyToTransfer > currentAccount.balance()) {
+                throw new Exception("Not enough money!");
+            } else {
+                bank.doTransfer(currentAccount.card().cardNumber(),
+                        receiver,
+                        moneyToTransfer);
+
+                System.out.println("Success!");
+            }
+        } catch (InputMismatchException | IllegalArgumentException e) {
+            System.out.println("Invalid input. Please try again.");
+        } catch (Exception e) {
+            System.out.println(e.getMessage());
+        }
+    }
+
+    private void closeAccountOption() {
+        bank.removeAccount(currentAccount.id());
+        logOut();
+
+        System.out.println("\nThe account has been closed!");
+    }
+
+    private void logOutOption() {
+        logOut();
+        System.out.println("\nYou have successfully logged out!");
+    }
+
+    private void createAccountOption() {
+        Card card = bank.registerAccount();
+        System.out.printf("""
+                %nYour account has been created
+                Your card number:
+                %s
+                Your card PIN:
+                %s%n""", card.cardNumber(), card.PIN());
+    }
+
+    private void logInOption() {
+        System.out.println("Enter your card number:");
+        String cardNumber = scanner.next();
+
+        System.out.println("Enter your PIN:");
+        String PIN = scanner.next();
+
+        Optional<Account> account = bank.signIn(cardNumber, PIN);
+        if (account.isPresent()) {
```

Comment:
Optional?

Reply:
Replaced if-else block with ifPresentOrElse method of Optional